File: Blog-Project/web/API/Models/Post.cs
using System;
namespace API.Models
{
    public class Post
    {
        public int id { get; set; }
        public string title { get; set; }
        public string body { get; set; }
        public string author { get; set; }
        public DateTime created_at { get; set; }
    }
}

File: Blog-Project/mobile/blog_app/lib/models/Post.dart
class Post {
  int id = 0;
  String date = '';
 
 String author = '';
  String title = '';
  String body = '';

  Post(
      {required this.id,
      required this.author,
      required this.date,
      required this.title,
      required this.body});

  // factory
  Post.fromJson(Map<String, dynamic> json) {
    id = json["Id"];
    date = json["CreatedAt"];
    author = json["Author"];
    title = json["Title"];
    body = json["Body"];
  }
}
